tests/test_diag_data_dictionary_spec.py

```diff
@@ -293,11 +293,6 @@ def test_initialization(self):
         self.assertEqual(len(ddds.structures), 0)
         self.assertEqual(len(ddds.end_of_pdu_fields), 0)
 
-        self.assertEqual({x.short_name
-                          for x in ddds.all_data_object_properties},
-                         {x.short_name
-                          for x in (dtc_dop, dop_1, dop_2, table)})
```

Comment:
Why?

Reply:
This is a drive-by fix which I failed to catch when pruning this series for the PR. (Sorry for that. I can move it to a separate PR if you want.)

The reason for this is that I came to the conclusion that the `all_data_object_properties` attribute does more harm than good, i.e., it is terribly easy to forget adapting it to changes elsewhere and there are no real users within the codebase. (the only valid use case which I can come up with out of my head is resolving SNREFs to DOPs, but even for this that attribute would only be useful if such SNREFs encompass *all* DOP objects, not just structures, simple DOPs, environment datas, etc and I'm not sure if this is even possible because the short names of these object categories might be allowed to collide.)